modules/ln_glgui/ln_glgui.scm

```diff
@@ -36,6 +36,23 @@ OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 |#
 
+(define-macro (define-cond-expand-feature-value v)
+  (let ((v (eval v)))
+    `(define-cond-expand-feature ,v)))
+
+(define-cond-expand-feature-value
```

Comment:
This piece should go to [modules/config/config.scm#L54](https://github.com/part-cw/lambdanative/blob/master/modules/config/config.scm#L54). You already declared the `define-macro` there, so that can be removed.

Reply:
Would not work.  define-macro is per compilation unit in gambit.  (Some Scheme's can export them, but gambit is not among those.)

I consider to provide a patch to `gambc` instead.  However that will take some time.  In the meantime this could be a work-around.

Reply:
Okay, that is certainly another comment to put into the pull request description.